### DISCUSSION TITLE
Can the crawler be used with .NET?

### DISCUSSION THREAD

**Discussion by Tony20221** [apify/crawlee-python#287] [Q&A] [answered]
Can the crawler be used with .NET? Did someone create something to make it work with .NET?

**Comment by janbuchar** [comment]
Hello, we are not aware of anything like that. Could you explain your use case? Maybe there is a way around this.

**Reply by Tony20221** [reply]
I don't know Python. I know C#.

**Reply by janbuchar** [reply]
Understood. We are not planning to port crawlee to C# anytime soon. Perhaps https://github.com/sjdirect/abot could be of use to you?

**Reply by Tony20221** [reply]
I am aware of Abot library.
I was asking for ways to interface with the Python library from C#. There aren't any.

**Reply by janbuchar** [reply, marked as answer]
Correct, there is no way you could use the library without knowing Python.

**Reply by mtrunkat** [reply]
You can deploy the Python-based crawler to apify.com and then simply call it  as an API from your .NET code.
